infrastructure/ansible/roles/nginx/templates/etc_nginx_sites_enabled_default.j2

```diff
@@ -10,12 +10,26 @@ server {
     ssl_certificate_key /etc/letsencrypt/live/{{ inventory_hostname }}/privkey.pem; # managed by Certbot
 
     ssl on;
-    ssl_session_cache  builtin:1000  shared:SSL:10m;
     ssl_protocols  TLSv1.2 TLSv1.3;
     ssl_ciphers EECDH+AESGCM:EDH+AESGCM;
     ssl_prefer_server_ciphers on;
     access_log            /var/log/nginx/java_server.access.log;
 
+    ssl_prefer_server_ciphers on; 
+    ssl_dhparam /etc/nginx/dhparam.pem;
+    ssl_ecdh_curve secp384r1; # Requires nginx >= 1.1.0
```

Comment:
@RoiEXLab we should know the version of nginx that we are deploying, the commentary should not really be here, or the infrastructure should take care to inject the right values for the right versions.

Reply:
To this extent, we should change the apt install to install a specific version:
```
  apt:
    name: nginx
    state: present
```
It might be better to add a comment on where the documentation is for these parameters, and/or create a wiki documentation page or add a README to the nginx role to describe why and what these parameters are doing.

Reply:
That specific update is arguably out of scope for now, but it's something we shoudl do before we spin up new prod.

Reply:
I agree that the comments are not required here, but I don't think it's necessary to lock the nginx version, once we can be sure that all the options are available using the current version in the repository

Reply:
We can get an out of sync state without locking version. We may for example test one version now and production then picks up a different one. When upgrading, we likely want to be explicit about it and will want to set a version number at some point to initiate the upgrade. Otherwise, without a version number, we're essentially saying "any" version will do, and that is not the case. For utilities like 'curl', it's appropriate to get any version, for a "main" system component where it matters quite a bit, I suspect we are better off with a specific version.